lib/PuppeteerSharp/Puppeteer.cs

```diff
@@ -24,7 +24,7 @@ public static class Puppeteer
         /// <summary>
         /// The default flags that Chromium will be launched with.
         /// </summary>
-        public static string[] DefaultArgs => ChromiumLauncher.DefaultArgs;
+        public static string[] DefaultFlags => ChromiumLauncher.DefaultArgs;
```

Comment:
Why are we changing this?

Reply:
Hi @kblok this change is in accordance to enable 1721 
![image](https://user-images.githubusercontent.com/37590429/102716022-17e66180-42ff-11eb-8e54-5cc5030c3490.png)

shall we ignore this as well ?

Reply:
Is that property being used in a Test?
Maybe we can make it internal.

Reply:
Hi @kblok 
super making it internal is a fab way around thank you ... Done